pronto-rubocop.gemspec

```diff
@@ -33,7 +33,8 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_runtime_dependency('pronto', '~> 0.11.0')
-  s.add_runtime_dependency('rubocop', '>= 0.63.1', '< 1.0')
+  s.add_runtime_dependency('rubocop', '>= 0.63.1', '< 2.0')
+  s.add_development_dependency('appraisal')
```

Comment:
Can you remove `appraisal` as I think the current GitHub CI configuration is good enough that you don't need it? Will make this PR much smaller as well 🙂

Reply:
I think that it would be easier to run rspec and such in local environment with Appraisal, but changed to generate `Gemfile.local` in GitHub Actions.

Reply:
I guess you forgot to remove this line?

Reply:
Never mind, I merged and removed it separately in 54fa5f0cad3700c9abc235ada20a9b731dff398a.